content/overview/pricing/pricing-model.md

```diff
@@ -2,14 +2,14 @@
 menu:
   overview:
     parent: pricing
-title: What is the pricing model?
+title: How the pricing model is managed
```

Comment:
This is as much as anything an answer to any questions you have about how changes to the model will come about. Not sure it answers "what it is". I want to consider this point and try making it simpler but going to hold off for now.